pages/dug_12/4-shameless-plug.md

```diff
@@ -0,0 +1,619 @@
+---
+theme: @fs/repo/slidev-template/theme
+layout: cover
+background: /intro.png
+class: text-center
+
+---
+
+## &#x1F44B; DUG#12 Shameless Plug &#x1F44B;
+
+<center><img src="/@fs/repo/img/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px"></center>
+
+---
+
+<br>
+
+<center><img src="/@fs/repo/img/dug_11/shameless_plug.png" width="340"></center>
+
+<br>
+
+- 🌱 **Growth & Transparency:** Showcasing our evolution and commitment to open-source.
+- 📚 **Historical Record:** A resource for 3mdeb, future customers, and the privacy/security community.
+- 📈 **Explore Business Model:** Learn from our open-source firmware journey, including potential pitfalls.
+
+---
+
+## What we will talk about
+
+<br>
+
+- Hardware,
+- Services,
+- Pace Enterprise Training,
+- Dasharo Pro/Enterprise Package
+- Accessories
+- Everything available in 3mdeb shop: https://shop.3mdeb.com
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+---
+
+# Hardware
+
+---
+
+<center><img src="/@fs/repo/img/dug_9/shop_sdwire.png" width="700"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/shop/open-source-hardware/sdwire/](https://shop.3mdeb.com/shop/open-source-hardware/sdwire/)
+</center>
+
+<!--
+
+We are in the middle of restock, new batch available half of December 2025.
+
+-->
+
+---
+
+<center><img src="/@fs/repo/img/dug_9/shop_twonkie.png" width="600"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/shop/open-source-hardware/twonkie-usb-c-sniffer/](https://shop.3mdeb.com/shop/open-source-hardware/twonkie-usb-c-sniffer/)
+</center>
+
+<!--
+
+Around 30 remain in stock.
+
+-->
+
+---
+
+<center><img src="/@fs/repo/img/dug_12/shop_usbsniffer.png" width="600"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product/asrock-spc741d8-2l2t-bcm-dasharo-pro-full-build/](https://shop.3mdeb.com/product/asrock-spc741d8-2l2t-bcm-dasharo-pro-full-build/)
+</center>
+
+<!--
+
+Awaiting final production offer.
+
+-->
+
+---
+
+<center><img src="/@fs/repo/img/dug_12/shop_usbarmory.png" width="600"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product/usb-armory-mk-ii/](https://shop.3mdeb.com/product/usb-armory-mk-ii/)
+</center>
+
+<!--
+
+Q1 2026
+
+-->
+
+---
+
+<center>
+
+### PCI USB controller that enables device certification for Qubes OS
+</center>
+<!--
+
+Q1 2026 ?
```

Comment:
@Stanislaw-bnk 
I am not sure if this product will be launched in Q1'26, more likely in 1H'26.
We do not have this product in our shop yet, not even as "coming soon", but I would still leave the slide about this product.
Be sure to write what this product is for. This information:
1. can be found in Jakub's task description, 
2. in the links to similar products that are linked there.

Perhaps R&amp;D has already written this for you in the PR in the product catalogue, and if it is still not there, sketch it out from the sources I mentioned and ask Jakub to review it.